charts/observability-pipelines-worker/README.md.gotmpl

```diff
@@ -53,7 +53,7 @@ organization. This Secret is used in the manifest to deploy the Observability Pi
 export DATADOG_SECRET_NAME=datadog-secrets
 kubectl create secret generic $DATADOG_SECRET_NAME \
     --from-literal api-key="<DD_API_KEY>" \
-    --from-literal config-key="<DD_OP_CONFIG_KEY>"
+    --from-literal pipeline-id="<DD_OP_PIPELINE_ID>"
```

Comment:
```suggestion
```

Since you're moving away from the pipelineId/configKey from being a secret you can probably drop this?